pom.xml

```diff
@@ -1524,11 +1525,21 @@
                     <branchRate>79</branchRate>
                     <lineRate>97</lineRate>
                   </regex>
+                  <regex>
+                    <pattern>com.puppycrawl.tools.checkstyle.grammars.javadoc.JavadocParser</pattern>
+                    <branchRate>23</branchRate>
+                    <lineRate>27</lineRate>
+                  </regex>
+                  <regex>
+                    <pattern>com.puppycrawl.tools.checkstyle.grammars.javadoc.JavadocLexer</pattern>
+                    <branchRate>41</branchRate>
+                    <lineRate>55</lineRate>
+                  </regex>
                 </regexes>
               </check>
               <instrumentation>
                 <excludes>
-                  <exclude>com/puppycrawl/tools/checkstyle/grammars/javadoc/*.class</exclude>
+                  <exclude>com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser$*.class</exclude>
```

Comment:
comment over exclude is required to explain why we keep it without consideration.